src/frequenz/sdk/microgrid/_resampling.py

```diff
@@ -179,7 +179,11 @@ def _log_resampling_task_error(self, resampling_task: asyncio.Task[None]) -> Non
             resampling_task.result()
         except ResamplingError as error:
             for source, source_error in error.exceptions.items():
-                _logger.error("Error resampling source %s, removing source...", source)
+                _logger.error(
+                    "Error resampling source %s, removing source: %s",
+                    source,
+                    source_error,
+                )
```

Comment:
Do you think this is enough? Maybe we can show the full backtrace with:

```suggestion
                _logger.error(
                    "Error resampling source %s, removing source",
                    source,
                    exc_info=source_error,
                )
```